docs/remote-control.md

```diff
@@ -6,24 +6,24 @@ sidebar_label: Remote Control
 
 import Keys from '@site/src/components/key.ts';
 
-With the addition of an external Wireless Access Point, the console can
-be remote controlled using **WiFi** from an [Android](https://play.google.com/store/apps/developer?id=Avolites+Ltd) or [iOS](https://apps.apple.com/us/developer/avolites-ltd/id688791177#see-all/i-phone-apps) mobile device
```

Comment:
The IOS link doesn't seem to work for me. Removing the ending so it's just https://apps.apple.com/us/developer/avolites-ltd/id688791177 made it work.